File: docker-mongo-node-fastify-typescript-jest-joi/src/modules/film/film-interface.ts
export interface NewFilm {
  title: string;
  episode_id?: number;
  opening_crawl?: string;
  director?: string;
  producer?: string;
  release_date?: string;
  characters?: string[];
  planets?: string[];
  starships?: string[];
  vehicles?: string[];
  species?: string[];
  is_deleted?: boolean;
}

File: mongo-net-asp-csharp/modules/film/film-interface.cs
namespace csharp_asp.modules.film
{
    public interface IFilm
    {
        string _id {get; set;}
        string Title { get; set; }
        int? EpisodeId { get; set; }
        string? OpeningCrawl { get; set; }
        string? Director { get; set; }
        string? Producer { get; set; }
        string? ReleaseDate { get; set; }
        List<string>? Characters { get; set; }
        List<string>? Planets { get; set; }
        List<string>? Starships { get; set; }
        List<string>? Vehicles { get; set; }
        List<string>? Species { get; set; }
        Boolean IsDeleted { get; set; }
        DateTime CreatedAt { get; set; }
        DateTime UpdatedAt { get; set; }
    }
}
